File: pgm/ABC_C#/ABC_147_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        String S = Console.ReadLine();
        int ans = 0;
        for (int i = 0; i < S.Length / 2; i++)
            if (S[i] != S[S.Length - i - 1]) ans++;
        Console.WriteLine(ans);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_147_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        Dim ans As Integer = 0
        For i As Integer = 0 To Math.Floor(Len(S) / 2) - 1
            If S(i) <> S(Len(S) - i - 1) Then
                ans += 1
            End If
        Next
        Console.WriteLine(ans)
        Return
    End Sub
End Class
